lib/portage/dep/__init__.py

```diff
@@ -1489,17 +1489,7 @@ def __init__(self, forbid=False):
         def __init__(self, forbid_overlap=False):
             self.overlap = self._overlap(forbid=forbid_overlap)
 
-    def __new__(
-        cls,
-        s,
-        unevaluated_atom=None,
-        allow_wildcard=False,
-        allow_repo=None,
-        _use=None,
-        eapi=None,
-        is_valid_flag=None,
-        allow_build_id=None,
-    ):
+    def __new__(cls, s, *args, **kwargs):
```

Comment:
I'm not a Python expert. Could you explain this change?

Reply:
I changed this simply to clean up the code - these parameters are not used at all, and thus it doesn't seem to make sense to name them.

In fact, I'd prefer to get rid of them altogether, but that would be a _slightly_ bigger refacter, as e.g. [this test](https://github.com/gentoo/portage/blob/master/lib/portage/tests/dep/testAtom.py#L216) seems to think that these parameters actually do something.